docs/development.md

```
@@ -184,6 +184,35 @@ any other use of that instance. `--upstream` and the state switcher are
 mutually exclusive; passing it replaces the switcher strip with one naming
 the upstream instead.
 
+## The browser demo
+
+```shell
+npm install && npm run demo:build      # assembles demo-dist/
+npx http-server demo-dist              # any static server works
+```
+
+The public demo ([issue
+#199](https://github.com/mark-brannan/signalk-noaa-space-weather/issues/199))
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)